libs/matdbg/README.md

```diff
@@ -0,0 +1,258 @@
+# matdbg
+
+1. [User Instructions](#user-instructions)
+1. [Architecture Overview](#architecture-overview)
+1. [C++ Server](#c-server)
+1. [JavaScript Client](#javascript-client)
+1. [HTTP Requests](#http-requests)
+1. [WebSocket Messages](#websocket-messages)
+1. [Wish List](#wish-list)
+1. [Screenshot](#screenshot)
+1. [Material Chunks](#material-chunks)
+
+## User Instructions
+
+Simply launch any app that links against a debug build of a Filament, and point your web browser to
+http://localhost:8080.
+
+After selecting a material and shader variant, you can view the GLSL, MSL, and SPIR-V code that was
+generated by `matc` or `filamat`. If the editor has focus, you can navigate between materials and
+variants by holding **Ctrl** while pressing the arrow keys.
+
+You can also make modifications to GLSL or MSL, so long as the shader inputs and uniforms remain
+intact. After making an edit, click the `[rebuild]` button in the header or press **Cmd+S**. Note
```

Comment:
Cmd-S on macOS but Ctrl+S on other platforms I assume?

Reply:
Yep, thanks, added to the doc.